theano/tensor/xlogx.py

```diff
@@ -1,5 +1,5 @@
+from __future__ import division
 import numpy
-
```

Comment:
keep the empty line. Pep8 tell us that we need to separate the import from out project of the others. In fact, there should be 3 import section import from python default library, import of 3th party python library, import from this library.